egs2/librimix/diar1/run.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Jiatong Shi
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+#
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+
+set -e
+set -u
+set -o pipefail
+
+train_set=train
+valid_set=dev
+test_sets=test
+
+train_config=conf/train_diar.yaml
+decode_config=conf/decode_diar.yaml
+num_spk=2
```

Comment:
Can you show the possible settings here (may be `num_spk=2 # 2, 3`

Reply:
Added the comment as suggested.